<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mycamera</class>
 <widget class="QWidget" name="Mycamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>700</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>700</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame{
	border-image: url(:/pictur/bg_music.jpg);
}
#frame_2{
	
	background-color: rgba(0, 0, 0, 100);
}


QGroupBox
{
	background-color: rgba(255, 255, 255, 100);
	color: rgb(0, 0, 0);
border: 2px solid rgb(153, 230, 230);
border-radius:5px;
margin-top:2.1ex;
font-family:宋体;
font:bold 30px;
color: ;
	color: rgb(0, 247, 119);
}
 

QGroupBox::title
{
subcontrol-origin:margin;
subcontrol-position:top center;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>700</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>700</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>700</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>661</width>
       <height>391</height>
      </rect>
     </property>
     <property name="title">
      <string>监控区域</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QVideoWidget" name="widget" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>430</y>
       <width>491</width>
       <height>251</height>
      </rect>
     </property>
     <property name="title">
      <string>照片预览</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_picture">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>20</y>
       <width>221</width>
       <height>681</height>
      </rect>
     </property>
     <property name="title">
      <string>控制台</string>
     </property>
     <widget class="QPushButton" name="pushButton_camera">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>开启摄像头</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);</string>
      </property>
      <property name="text">
       <string>摄像头类型</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>150</y>
        <width>99</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_shutter">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>490</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>拍照</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_del">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>600</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_client">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>151</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>开启对方摄像头</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
